packages/components/divider/src/divider.ts

```diff
@@ -0,0 +1,17 @@
+import { buildProp } from '@element-plus/utils/props'
+
+import type { ExtractPropTypes } from 'vue'
+
+export const dividerProps = {
+  direction: buildProp({
+    type: String,
+    values: ['horizontal', 'vertical'] as const,
```

Comment:
Later in the future we can get these out since I knew many components use this for direction.